scripts/setup-systemd

```diff
@@ -16,4 +16,25 @@ Restart=always
 [Install]
 WantedBy=multi-user.target" > /etc/systemd/system/grist.service
 
+echo "[Unit]
+Description=Upgrade Docker Compose services for Grist
+After=network.target docker.service
+Requires=docker.service
+
+[Service]
+Type=oneshot
+WorkingDirectory=/home/grist
+ExecStart=/home/grist/bin/upgrade-grist"  > /etc/systemd/system/grist-upgrade.service
+
+echo "[Unit]
+Description=Run Docker Compose upgrade for Grist weekly
+
+[Timer]
+OnCalendar=Sun *-*-* 00:00:00
+RandomizedDelaySec=2h
+Persistent=true
```

Comment:
I think this might cause an instant restart when the timer is enabled - is that intentional?

Reply:
Testing, no, the timer isn't triggered the first time. What `Persistent=true` should do is run the upgrade at the next available opportunity in case the system was unavailable (e.g. powered down) during the scheduled time. 
It's not unusual to turn off cloud instances during the weekend as a cost-saving measure, so I think I'll leave this option in.